src/digraph.rs

```diff
@@ -547,6 +547,25 @@ impl PyDiGraph {
         }
     }
 
+    /// Return a list of indices of all directed edges between specified nodes
+    ///
+    /// :returns: A list of all the edge indices connecting the specified start and end node
+    /// :rtype: EdgeIndices
+    #[pyo3(text_signature = "(self)")]
```

Comment:
The text signature doesn't match the actual function signature and this is the root cause of the mypy stub failure. Removing the text signature should fix it (as pyo3 should auto generate it now if you don't provide it).
```suggestion
```